extensions/ql-vscode/src/language-support/index.ts

```diff
@@ -1,3 +1,4 @@
+export * from "./contextual";
```

Comment:
Did you mean to export everything from `contextual` to the outside world? I would have thought that some of it is only internal to `language-support` (e.g. the `file-range-from-uri` stuff).

Reply:
Discussed synchronously, fixed